README.md

```diff
@@ -1,76 +1,115 @@
-                         both together in a `<name>.contract` file
-    generate-metadata    Command has been deprecated, use `cargo contract build` instead
-    check                Check that the code builds as Wasm; does not output any
-                         `<name>.contract` artifact to the `target/` directory
-    test                 Test the smart contract off-chain
-    deploy               Upload the smart contract code to the chain
-    instantiate          Instantiate a deployed smart contract
-    help                 Prints this message or the help of the given subcommand(s)
-```
+You can always use `cargo contract help` to print information on available
+commands and their usage.
 
-## `build` requires the `nightly` toolchain
+For each command there is also a `--help` flag with info on additional parameters,
+e.g. `cargo contract new --help`.
 
-`cargo contract build` must be run using the `nightly` toolchain. If you have 
-[`rustup`](https://github.com/rust-lang/rustup) installed, the simplest way to do so is `cargo +nightly contract build`.
-To avoid having to add `+nightly` you can also create a `rust-toolchain` file in your local directory containing 
-`nightly`. Read more about how to [specify the rustup toolchain](https://github.com/rust-lang/rustup#override-precedence).
+##### `cargo contract new my_contract`
 
-### Note 
+Creates an initial smart contract with some scaffolding code into a new
+folder `my_contract` .
 
-The latest version of `cargo-contract` supports all nightlies after `2020-07-30`, because of a change in the directory
-structure of the `rust-src` component. 
+The contract contains the source code for the [`Flipper`](https://github.com/paritytech/ink/blob/master/examples/flipper/lib.rs) 
+contract, which is about the simplest "smart" contract you can build ‒ a `bool` which gets flipped
+from `true` to `false` through the `flip()` function.
 
-## Features
+##### `cargo +nightly contract build`
 
-The `deploy` and `instantiate` subcommands are **disabled by default**, since they are not fully stable yet and increase the build time.
+Compiles the contract into optimized WebAssembly bytecode, generates metadata for it,
+and bundles both together in a `<name>.contract` file, which you can use for
+deploying the contract on-chain.
+
+`cargo contract build` must be run using the `nightly` toolchain. If you have
+[`rustup`](https://github.com/rust-lang/rustup) installed, the simplest way to
+do so is `cargo +nightly contract build`.
+
+To avoid having to add `+nightly` you can also create a `rust-toolchain` file in your local
+directory containing `nightly`. Read more about how to [specify the rustup toolchain](https://github.com/rust-lang/rustup#override-precedence).
+
+##### `cargo contract check`
+
+Checks that the code builds as WebAssembly. This command does not output any `<name>.contract`
+artifact to the `target/` directory.
+
+
+#### Deploy and Instantiate
```

Comment:
Actually can you remove this whole section, these commands are currently completely broken (not compatible with any recent version of the contracts pallet).

Will add it back once these are working again